src/asm/rgbasm.5

```diff
@@ -752,12 +752,12 @@ or
 .Ic FRAGMENT
 modifiers, as described below.
 .Ss Unionized Sections
-When you're tight on RAM, you may want to define overlapping memory allocations, as explained in the
+When you're tight on RAM, you may want to define overlapping static memory allocations, as explained in the
 .Sx Unions
 section.
 However, the
 .Ic UNION
-keyword only works within a single file, which prevents e.g. defining temporary allocations on a single memory area across several files.
+keyword only works within a single file, which prevents e.g. defining overlapping static allocations on a single memory area across several files.
```

Comment:
The intent was pointing out the common use case of "temporary variables over a single memory region", hence the current wording. Can it be improved?

Reply:
How about this:

"However, a UNION only works within a single file, so it can't be used e.g. to define temporary variables across several files, all of which use the same statically allocated memory."